rfcs/text/0030/terminal.yml

```diff
@@ -0,0 +1,86 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+- name: terminal
+  title: Terminal
+  group: 2
+  short: The terminal info for an interactive session.
+  description: >
+    These values define how applications such as editors and shells have configured the terminal and how they decide how to display things like colors and highlighting. Important aspects of the c_* bit flags fields include whether UTF-8 is supported and if echoing of input is enabled. Terminal row count, column count, TERM environment variable and other fields can be a user fingerprint as well. See “man 5 terminfo” and “man 2 ioctl_tty” for more information.
+
+    Only present in the top level process (not process.parent etc.) and only when that process's session has a controlling TTY (ie the process is part of an interactive session)
+  type: group
+  reusable:
+    top_level: false
+    expected:
+      - process
```

Comment:
Should terminal information be in the process, or should it be a top level field of an output event?